assets/src/scripts/components/hydrograph/tooltip.js

```diff
@@ -64,13 +64,9 @@ const tooltipPointsSelector = memoize(tsKey => createSelector(
     }
 ));
 
-const updateTooltipText = function(text, {cursorPoints, qualifiers, unitCode}) {
+const getTooltipText = function(datum, qualifiers, unitCode) {
```

Comment:
Would it be worthwhile to add a test for this function? It's previous variant didn't have a test, so I think it's a show stopper, but I think it would be good to add tests for functions if we're touching them and they don't already have them.

Reply:
I'm inclined to keep it as-is since the code is being tested higher up the chain.